### DISCUSSION TITLE
Adrenaline is not very usable

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#13715] [Unstable feedback]
Adrenaline has been granted a functionality after many years, yet the current implementation is not very usable.
The problem is twofold: a) the duration is exceedingly short so as to be near useless, and b) the chem addiction penalty is exceedingly high so as to discourage use altogether.

5 seconds is too short to really do anything in most cases. You might be able to finish off an enemy, but you fall unconscious seconds later. You can't retreat, it doesn't grant any defensive or offensive bonus. There are really only two use cases that may actually change the outcome of a situation, that being a medic using it just before going down so they can use medicines on themselves to get back above critical before the adrenaline runs out (which is somewhat niche in itself, since if the medic has meds on them, then they can usually heal themselves before going down)
The second use case is completely unrelated to combat, but the only actual use we've found for it, and that's using adrenaline on a suffocating person to make them wake up and start breathing if there aren't any crew members with high medical skill.

Adrenaline has to be applied before going down, so some duration is always wasted. It's not possible to predict exactly when you are about to fall unconscious, so either you use it too early and only get 2-3 seconds of functionality, or you use it too late and die anyway. This is a problem it shares with most of the other drugs - you don't know when you'll need it, so more often than not it's wasted, or not used in time.

The functionality of being immune to falling unconscious inspires the idea of a last ditch effort to win a fight, but the duration is too short to do much, and it's an exceptionally bad idea to use multiple for the next reason.

The chem addiction penalty is extremely high. It's worse than all the other drugs by a large margin, and using more than one is really dangerous, since chem addiction has no cure. Using just two is 60 chem addiction, which takes _40 minutes_ to wear off, and is lethal if you don't get to the outpost clinic. Lastly, the chem addiction isn't even lower with high medical skill, so there's no incentive for medics to use them compared to non-medics.

Overall, adrenaline isn't very useful in combat situations and is suicidal to use multiple of. It would be nicer if it wasn't so time sensitive and gave enough time to finish a fight (as an example, give an affliction that lasts for one minute, and if you fall unconscious during that time, you'll get 5-10 seconds of consciousness before falling, thus the timing on using the adrenaline is not so pressing, and you get the full duration and are less likely to waste it.)
If it's intended not to be a combat drug, and is supposed to only revive suffocating patients, then it shouldn't have such an obscene chem addiction cost, and probably doesn't need to last as long.

**Comment by ubertpendragon** [comment]
In short increase duration to 10 reduce chemaddiction to 1

**Comment by Cherman0** [comment]
I assumed the intention was to revive unconscious people without CPR, not for premedding yourself to be basically double health in a fight. The chem addiction could probably get reined in somewhat, though I believe the intention is that you only ever have to use one on somebody. Adrenaline is not an addictive substance irl (the conception of an "adrenaline junkie" is not actually related to adrenaline) for reference.

**Reply by NotWendy** [reply]
Reviving people without CPR so they can breathe on their own is the most obvious use of the adrenaline, which is kind of sad, because that used to be liquid oxygenite's niche. (Which it also filled badly by having high skill requirement.)

**Reply by Cherman0** [reply]
I never got the impression that liquid oxygenite was meant to be something that was actually a good idea to inject into people, but that's getting a bit off topic.
